teaching.md

```
@@ -0,0 +1,87 @@
+
+<h2>Linguistics & K-12</h2>
+As of 2025, I've been exploring the question of what insights from theoretical linguistics can be applied in pre-university classrooms, to inform & transform (language) pedagogy — a line of work that is relatively underexplored in both linguistics and education spaces. Below are some of my recent output for K-12 students & teachers:
+<ul>
+ <li>An <a href="https://drive.google.com/file/d/1SO8hN3rNEPL7aJ77xHhtBwaUNfxquW5J/view?usp=sharing">informational guide</a> for teachers that introduces ideas for bringing linguistics into the classroom.</li>
+ <li>A <a href="https://docs.google.com/presentation/d/1n89c2lWBXurFI8sFrZLGKluargDcVqds5XOlU25Y5z8/edit?usp=drive_link">lesson on the linguistics of word-level stress</a>, delivered to a 7th-grade English class to supplement instruction on poetry. Special thanks to Beth Keyser for this opportunity.
+</li>
+ <li>A set of introductory linguistics crashcourse materials, taught to high school students at <a href="https://docs.google.com/presentation/d/1MEuvq0Itn0YTuoVcWvOu1Uw7jkXBNtdk_IsPYC5ogyc/edit?usp=drive_link">MIT Splash 2025</a> and <a href="https://docs.google.com/presentation/d/1Gn_4Kod_57-sjvaWnCwkUk4NSlypf9IkiyaveM-aTs8/edit?usp=sharing">Harvard HWeek 2025</a>, with the latter having a Mandarin-English bilingual component.</li>
+ <li>Pop-up courses on <a href="https://docs.google.com/presentation/d/1Gig4xLUvgAiH9aoDImZQdctZHjPJwpb9VxAL_OQmuFE/edit?usp=sharing">phonology</a> and <a href="https://docs.google.com/presentation/d/1dset725RBrejbBzvNmchTbNuxRKJx4bTDgAn9VSicuM/edit?usp=sharing">syntax</a> taught to high school students at MIT HSSP 2026. Special thanks to Dr. Maya Honda (MIT Linguistics) for this opportunity.</li>
+</ul>
+
+<br><img style="margin-top: -20px; display: block; margin-left: auto; margin-right: auto; height: 80%; width: 80%; border-radius: 10px" src="/assets/img/ling-classroom.jpg">
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)